src/Parser/Rogue/Subtlety/Modules/Items/T21_2P.js

```diff
@@ -0,0 +1,24 @@
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import ResourceTypes from 'common/RESOURCE_TYPES';
+import Analyzer from 'Parser/Core/Analyzer';
+
+class T21_2P extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+		spellUsable: SpellUsable,
+  };
+
+  on_byPlayer_spendresource(event){
+    const spent = event.resourceChange;
+    if(event.resourceChangeType !== ResourceTypes.COMBO_POINTS) return;
+
+    if(this.combatants.selected.hasBuff(SPELLS.SUB_ROGUE_T21_2SET_BONUS.id 
+      && this.spellUsable.isOnCooldown(SPELLS.SYMBOLS_OF_DEATH.id))){
```

Comment:
Use on_initialized() { } to check for tier bonus' so it doesn't show if the person doesn't have it equipped

Reply:
Nothing is showing up for this atm anyway, but  changed it for future.